go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/docker/docker v20.10.8+incompatible
```

Comment:
go.mod still needs to be updated to require 1.18. 

BTW, in the go.mod we cannot specify 1.18.3 as in the CI, so I was wondering if we should change the CI also to use 1.18 instead of 1.18.3. WDYT?

Reply:
> go.mod still needs to be updated to require 1.18.

Oh, I've broken my rebase. Let me fix this.

> BTW, in the go.mod we cannot specify 1.18.3 as in the CI, so I was wondering if we should change the CI also to use 1.18 instead of 1.18.3. WDYT?

I think we should not: the go.mod version is for backward compatibility (they don't change the semantics between minor versions) but we can still take the last Go version.